tests/v4l2encode.cpp

```diff
@@ -147,23 +148,23 @@ bool feedOneInputFrame(int fd, int index = -1 /* if index is not -1, simple enqu
 
 bool writeOneOutputFrame(uint8_t* data, uint32_t dataSize)
 {
-    static FILE* fp = NULL;
+    static std::ifstream m_ofs;
```

Comment:
I noticed just now that the existing code had the file marked as 'static' and I hadn't added that. Not a good combo when you open with 'trunc' enabled.

Reply:
This should be a `std::ofstream`... and static just means it'll stay open until the program ends.

Reply:
hmmm.. yes I see now that it is closed explicitly on isOutputEOS.  I'm not sure what the original author's intention is here... but your refactor looks logically equivalent (with std::ofstream change of course)

Reply:
Yeah, I don't understand why some of the things in this file were done the way they were, but I can roll with it. :)